cmd/create/bomupload.go

```diff
@@ -0,0 +1,56 @@
+package create
+
+import (
+	"log"
+	"os"
+
+	mctl "github.com/metal-toolbox/mctl/cmd"
+	"github.com/metal-toolbox/mctl/internal/app"
+	"github.com/spf13/cobra"
+)
+
+// Create Bom informations.
+type uploadBomFileFlags struct {
+	// Xlsx file containing one or multiple boms information.
+	bomXlsxFile string
+}
+
+var (
+	flagsUploadBomFileFlags *uploadBomFileFlags
+)
+
+var uploadBomFile = &cobra.Command{
+	Use:   "uploadbom",
```

Comment:
This has a weird sound in English to me. What about `create bom` and (like you do), call out specifically in the help that this is uploading a BOM to BOMService?

Reply:
Thanks! Will include it in the the follow-up PR.